cmd/agent/daemon/app/app.go

```diff
@@ -69,7 +73,7 @@ func (c Config) Proto() *castaipb.AgentConfig {
 		HostCgroupsDir:        c.HostCgroupsDir,
 		MetricsHttpListenPort: int32(c.MetricsHTTPListenPort),
 		State: &castaipb.AgentStateControllerConfig{
-			EventsSinkQueueSize:          int32(c.State.EventsSinkQueueSize),
+			//EventsSinkQueueSize:          int32(c.State.EventsSinkQueueSize),
```

Comment:
Should this line be commented out?

Reply:
Yep, need to remove it and make exporters queue configurable, this is now not used.